virtualenv.py

```diff
@@ -1478,6 +1479,13 @@ def install_python_config(home_dir, bin_dir, prompt=None):
     for name, content in files.items():
         make_exe(os.path.join(bin_dir, name))
 
+def install_shared(bin_dir, symlink=True):
+    current_shared = os.path.realpath(join(os.path.dirname(sys.executable), "..", "lib"))
```

Comment:
Is this going to exist on all platforms (MacOS, Windows, Linus) / most distros? This function only seems to care about nixxy platforms AFAICT.

Reply:
I think my original idea is that its existence does not matter, because if it does not exist, the next `glob.glob` would return empty list then nothing will be copied.